meetings/2024-09-11.md

```diff
@@ -44,23 +44,22 @@
+* MLG: Once I fetch/load the first dataset (profile), I can use mixins. With mixin I have access to methods I can call on the dataset to access to data in an easy way. With only few lines I can easily query the pod. I can also show you another example. A bit of code I'm working on for Startin'Blox which shows the usage of the TypeIndex mixin: I can get the TypeIndex by simply call the `getPublicTypeIndex` method. Then I can directly access to a registration instance with the dedicated method `getRegistratedInstanceForClass`. 
 * eP: Can you show us how you define those mixins.
-* MLG: Mixin are very easy to define. It is just a mixin function in TS. You just define have to define some methods which you want to add to your dataset. You also have to export a factory to create and load easily instances of those mixins (an helper is available).
-* eP: Do you have an example of using multiple mixins on the same resource
-* MLG: The SolidWebID mixin uses the WebIDMixin and the DatasetMixin. The DatasetMixin is the base mixin which adds the URI of the dataset.
+* MLG: Mixin are very easy to define. It is just a mixin function in TS in which you define some methods you want to add to your dataset. You also have to export a factory to create and load instances of your mixin (an helper is available).
+* eP: Do you have an example of using multiple mixins on the same resource?
+* MLG: Yes, for instance the SolidWebID mixin uses the WebIDMixin and the DatasetMixin. The DatasetMixin is the base mixin which adds the URI of the dataset.
 * eP: Could you use WebID mixin and Vcard mixin together for example?
-* MLG: Yes, mixins can be mixed like wanted.
+* MLG: Yes, mixins can be mixed like wanted to add an unlimited number of methods to manipulate the data.
 * eP: How do you compare it to LDO and the class based approach used by Noel?
-* MLG: LDO only defines properties to access underlying data (defined from a shape). If you want to add methods to LDO objects you have to do it by yourself. Semantizer can be used to do so. Semantizer adds mixin, which supports RDFJS datasets and all the RDF data model (blank nodes also).
-* MLG: Semantizer can be used to add some methods to LDO objects. For example to lint, add logic or compute some values.
+* MLG: LDO only defines properties to access underlying data (defined from a shape). If you want to add methods to LDO objects you have to do it by yourself. Semantizer can be used to do so as it can work with any RDFJS compatible dataset. Semantizer can be used to add some methods to LDO objects for example to lint, add logic or compute some values. Semantizer also supports blank nodes (which the lib of Noël does not).
```

Comment:
```suggestion
* MLG: LDO only defines properties to access underlying data (defined from a shape). If you want to add methods to LDO objects, you have to do it by yourself. Semantizer can be used to do so, as it can work with any RDFJS compatible dataset. Semantizer can be used to add some methods to LDO objects; for example, to lint, add logic, or compute some values. Semantizer also supports blank nodes (which the lib of Noël does not).
```